vcd/resource_vcd_vapp_vm_boot_options_test.go

```diff
@@ -0,0 +1,246 @@
+//go:build vapp || vm || ALL || functional
+
+package vcd
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+)
+
+func TestAccVcdVAppVmBootOptions(t *testing.T) {
+	preTestChecks(t)
+	var (
+		vapp     govcd.VApp
+		vm       govcd.VM
+		vappName string = t.Name()
+		vmName   string = t.Name() + "VM"
+	)
+
+	if testConfig.VCD.ProviderVdc.StorageProfile == "" || testConfig.VCD.ProviderVdc.StorageProfile2 == "" {
+		t.Skip("Both variables testConfig.VCD.ProviderVdc.StorageProfile and testConfig.VCD.ProviderVdc.StorageProfile2 must be set")
+	}
+
+	var params = StringMap{
+		"Org":         testConfig.VCD.Org,
+		"Vdc":         testConfig.VCD.Vdc,
```

Comment:
Would be better to create test VMs in NSX-T VDC `testConfig.Nsxt.Vdc` as we're going to deprecate and remove NSX-V support at some time.

Reply:
Agree. Updated.